### PR TITLE
photo thumbnails added

### PR DESCRIPTION
**Pull request by devpool007**

### CODE REVIEW

**Comment by devpool007** [comment]
Damn I see this has already been implemented and in better way guess I will try improving my method.

File: AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift [hunk 1]
```diff
@@ -14,15 +14,53 @@
...
 import Foundation
 
 struct ApiConfig {
     
-    static let BASE_URL =                       "https://example.com"
-    static let PROXY_URL =                       "https://example.com"
+    static let BASE_URL =                       "https://api.amahi.org"
+    static let PROXY_URL =                      "https://pfe.amahi.org"
+    
+    private static let CLIENT_ID =              "0714493ef3f647209dfe0574fbc3b426220714aaedfd6c007cd54f33e4a0d4bb"
```
Comment (codedentwickler) [inline AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: You should not have pushed this file to your PR. Now out secret Api Configurations are out. 
Please delete this PR so it is not accessible to the public.

Comment (devpool007) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: sure sorry for this

Comment (devpool007) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: how to delete this pr?

Comment (ThisIsNSH) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: git checkout devansh_gsoc
git reset --soft HEAD~1
git checkout AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
git add .
git commit -m "commit name"
git push -f origin devansh_gsoc

**Comment by ThisIsNSH** [comment]
I can still see the Config file. Remove config file from the commit using the git code commented above

File: AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift (hunk 1 above)
Comment (devpool007) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: ok let me try this, if it work I will update you, how do you have so much knowledge its cool tbh

Comment (devpool007) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: This is not working :(

**Comment by devpool007** [comment]
> I can still see the Config file. Remove config file from the commit using the git code commented above

Its not working I am getting this : 

fatal: 'orign' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

File: AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift (hunk 1 above)
Comment (ThisIsNSH) [reply AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift]: make sure you are on same branch.

**Comment by ThisIsNSH** [comment]
whats your remote?
check it 
git remote -v

**Comment by devpool007** [comment]
> whats your remote?
> check it
> git remote -v

origin	https://github.com/devpool007/amahi_ios.git (fetch)
origin	https://github.com/devpool007/amahi_ios.git (push)

**Comment by ThisIsNSH** [comment]
oh 
you made spelling mistake

its origin not orign

**Comment by devpool007** [comment]
> oh
> you made a spelling mistake
> 
> its origin not orign

Ok I pushed is it gone?

**Comment by ThisIsNSH** [comment]
git checkout devansh_gsoc
git reset --soft HEAD~1

* open AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
* make it as it was before (without credentials)

git add .

git commit -m "commit name"
git push -f origin devansh_gsoc

**Comment by ThisIsNSH** [comment]
if this doesnt work then try this: (will work 100%)

git checkout devansh_gsoc
git reset --soft HEAD~1
git status

git add **file names which appear in git status, add them all. files to be added are 9 only** 
*make sure you dont add apiconfig here*

git commit -m "commit name"
git push -f origin devansh_gsoc

**Comment by devpool007** [comment]
> git checkout devansh_gsoc
> git reset --soft HEAD~1
> 
> * open AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
> * make it as it was before (without credentials)
> 
> git add .
> 
> git commit -m "commit name"
> git push -f origin devansh_gsoc

It WORKED 👍  Thnx a lot man :D  Where did you learn all this?

**Comment by ThisIsNSH** [comment]
Good 👍  we are done 
@codedentwickler please remove your comment from this pr. it shows the id

**Comment by ThisIsNSH** [comment]
> > git checkout devansh_gsoc
> > git reset --soft HEAD~1
> > 
> > * open AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
> > * make it as it was before (without credentials)
> > 
> > git add .
> > git commit -m "commit name"
> > git push -f origin devansh_gsoc
> 
> It WORKED 👍 Thnx a lot man :D Where did you learn all this?

little practice. welcome buddy

**Comment by devpool007** [comment]
> > > git checkout devansh_gsoc
> > > git reset --soft HEAD~1
> > > 
> > > * open AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
> > > * make it as it was before (without credentials)
> > > 
> > > git add .
> > > git commit -m "commit name"
> > > git push -f origin devansh_gsoc
> > 
> > 
> > It WORKED 👍 Thnx a lot man :D Where did you learn all this?
> 
> little practice. welcome buddy

cool :) u saved us

**Pull request closed without merging**